internal/storage/v2/clickhouse/schema/schema.sql

```diff
@@ -10,6 +10,12 @@ CREATE TABLE IF NOT EXISTS spans (
     status_message String,
     duration Int64,
 
+    bool_attributes Nested (key String, value Bool),
+    double_attributes Nested (key String, value Float64),
+    int_attributes Nested (key String, value Int64),
+    str_attributes Nested (key String, value String),
+    bytes_attributes Nested (key String, value String)
```

Comment:
If the value is string do we need a separate field? We will have to invent some kind of naming convention to distinguish plain string attributes from serialized json, and if we do we can just use the same type for bytes.

Reply:
@yurishkuro I see. And then would we remove the prefix when serving the traces?

Reply:
Sounds good! I can do this in the next PR.